README.md

```diff
@@ -1,4 +1,4 @@
-# Rust Markdown Parsing Benchmarks
+# Rust JSON Parsing Benchmarks
```

Comment:
> Benchmarks appear to be testing json, not markdown.

Its neither, its benchmarks of parsers, with a json parser being one way of testing it.

Thanks for pointing this out though! I've got it fixed now.